async-query-core/src/main/java/org/opensearch/sql/spark/parameter/SparkSubmitParameters.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.parameter;
+
+import static org.opensearch.sql.spark.data.constants.SparkConstants.DEFAULT_CLASS_NAME;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import lombok.Setter;
+
+/** Define Spark Submit Parameters. */
+public class SparkSubmitParameters {
```

Comment:
Moved from: 
async-query-core/src/main/java/org/opensearch/sql/spark/asyncquery/model/SparkSubmitParameters.java
And builder was extracted as separate class.